src/components/About/about.tsx

```diff
@@ -0,0 +1,45 @@
+import './about.css';
+
+export const About = () => {
+  return (
+    <div className="about-wrapper">
```

Comment:
Why it's not a ".page-wrapper" it could be common

Reply:
as you wish